charts/shoot-core/components/values.yaml

```
@@ -49,7 +49,7 @@ shoot-info:
   maintenanceEnd: 220000+0000
   extensions: shoot-dns-service,foo-bar
 vpn-shoot:
-  enabled: true
+  enabled: false
```

VPN-shoot should still be "the default"

```suggestion
  enabled: true
```

---

but then we would have to explicitly turn it off in code when the featuregate is on.

---

Hm okay, right. Fine then, thanks.